### ISSUE TITLE
I have a question about Train, Test.

### ISSUE THREAD

**Issue by cslee99** [jahongir7174/YOLOv5-tf#9]
When training, try with the file below.
    -IMAGES, LABELS, train.txt
When testing, try with the following file.
    -IMAGES, val.txt

In general, training is classified into three categories: Training set, Validation set, and Test set.
Is YOLOv5 configured as follows?
Train = Training set + Validation set
Test = Test set
In the case of Train, is the training set and validation set automatically classified and trained?
If this is correct, what is the ratio?

**Comment by jahongir7174** [comment]
dividing the dataset into parts is up to you. if you add your validation or test data into train.txt, then it uses your added data in training otherwise it does not
